x-pack/plugins/lens/public/pie_visualization/suggestions.test.ts

```diff
@@ -784,7 +784,8 @@ describe('suggestions', () => {
       ).toHaveLength(0);
     });
 
-    it('mosaic type should be added only in case of 2 groups', () => {
+    /** The following test should be activated when Experimental flag will be removed from the Mosaic chart **/
+    it.skip('mosaic type should be added only in case of 2 groups', () => {
```

Comment:
We could turn these into tests to check whether the hidden suggestions are correct, what do you think?

Reply:
just the tests below check that and since we always have `hide: true` it doesn't really make much sense to keep both.